src/Illuminate/Contracts/Cache/Repository.php

```diff
@@ -37,7 +37,7 @@ public function pull($key, $default = null);
      *
      * @param  string  $key
      * @param  mixed   $value
-     * @param  \DateTime|float|int  $minutes
+     * @param  \DateTimeInterface|\DateInterval|float|int  $minutes
```

Comment:
We could probably have `mixed` there, but I've let this on purpose; this can be modified before `5.5` release if necessary.